octoprint_discordremote/__init__.py

```diff
@@ -144,6 +146,7 @@ def get_settings_defaults(self):
             'bottoken': "",
             'channelid': "",
             'allowedusers': "",
+            'baseurl': self.get_ip_address(),
```

Comment:
@OctopusProteins I think default value should be None, and the logic that uses baseurl should just swap None/"" with self.get_ip_address()

Reply:
I can't access the ```get_ip_address()``` function from the Command class, unless I'm missing something...?

Reply:
Fixed by 1007480